.github/workflows/release.yml

```diff
@@ -0,0 +1,57 @@
+name: release
+on:
+  push:
+    tags:
+      - '*'
```

Comment:
This workflow is usually quite robust and does not fail, but I'd recommend in any case to run this also for PRs, limiting just the last step to upstream ([example](https://github.com/diegoferigo/gazebo-yarp-synchronizer/blob/568c10f65826c29a100137c6f8b3815157f7d4c7/.github/workflows/python.yml#L109-L116)).

Also, I find inspecting the dist folder quite useful (the step before the last).

Reply:
Thanks for pointing this out, I'll update it :+1:.